app/assets/stylesheets/self-service-time-to-pay.css

```diff
@@ -242,10 +242,10 @@ input.accountName{
     height: 1.9em;
 }
 @media (min-width: 48.0625em){
-    #id_summary ..govuk-summary-list__key {
+    #id_summary .govuk-summary-list__key {
```

Comment:
intellij said this was a syntax error - is this ok now @tp10?

Reply:
Yes, a double dot is a syntax error.  The new version is valid CSS.